packages/angular/cli/src/commands/add/cli.ts

```diff
@@ -107,9 +105,9 @@ export class AddCommandModule
 
   // eslint-disable-next-line max-lines-per-function
   async run(options: Options<AddCommandArgs> & OtherOptions): Promise<number | void> {
-    const { root, logger, packageManager } = this.context;
+    const { logger, packageManager } = this.context;
     const { verbose, registry, collection, skipConfirmation } = options;
-    await ensureCompatibleNpm(root);
+    await packageManager.ensureCompatibleNpm();
```

Comment:
nit: maybe just `ensureCompatibility` now in case we ever expand the checks?

Reply:
Agree